### DISCUSSION TITLE
How do I import the tabler icons?

### DISCUSSION THREAD

**Discussion by NatSo1** [ng-icons/ng-icons#53] [Q&A] [answered]
I've followed the instructions. The compiler is saying the icons aren't in the location. I'm using the NG modules instructions. 
Where is the actual import location of the icons for this package? This is what I wrote:

import { NgIconsModule} from '@ng-icons/core';
import { tablerCircleLetterL } from '@ng-icons/tabler-icons/esm2020/ng-icons-tabler-icons.mjs';
import { tablerCircleLetterP } from '@ng-icons/tabler-icons/esm2020/ng-icons-tabler-icons.mjs';

**Comment by ashley-hunter** [comment, marked as answer]
Hi, 

It should just be as simple as importing the icons like this:

```ts
import { BrowserModule } from '@angular/platform-browser';
import { NgModule } from '@angular/core';
import { NgIconsModule } from "@ng-icons/core";
import {
  tablerCircleLetterL,
  tablerCircleLetterP
} from "@ng-icons/tabler-icons";
import { AppComponent } from './app.component';

@NgModule({
  declarations: [
    AppComponent
  ],
  imports: [
    BrowserModule,
    NgIconsModule.withIcons({
      tablerCircleLetterL,
      tablerCircleLetterP
    })
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
```

You should not need to deep import, just from the package:

[See Example CodeSandbox](https://codesandbox.io/p/sandbox/smoosh-violet-ir2lvi?file=%2Fsrc%2Fapp%2Fapp.module.ts&selection=%5B%7B%22endColumn%22%3A23%2C%22endLineNumber%22%3A5%2C%22startColumn%22%3A23%2C%22startLineNumber%22%3A5%7D%5D)

If that is not the case, let me know in case there is a bug, thanks!
